pkg/cloud/services/autoscaling/autoscalinggroup.go

```diff
@@ -303,6 +304,19 @@ func (s *Service) UpdateASG(scope *scope.MachinePoolScope) error {
 	return nil
 }
 
+func (s *Service) StartASGInstanceRefresh(scope *scope.MachinePoolScope) error {
+	input := &autoscaling.StartInstanceRefreshInput{
```

Comment:
Would it be worth exposing the preferences (i.e. instance warmup and min health percentage)? Might be useful to users that want more control. WDYT?

Reply:
If we can do it now, that'd be preferable.

Reply:
Do you mean exposing them in CRD?

Reply:
Sorry, yes in the CRD and then using them here.